protected/humhub/modules/content/widgets/WallCreateContentMenu.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2022 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\content\widgets;
+
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\content\components\ContentContainerActiveRecord;
+use humhub\modules\content\widgets\stream\WallStreamEntryWidget;
+use humhub\modules\ui\menu\MenuLink;
+use humhub\modules\ui\menu\widgets\Menu;
+use Yii;
+
+/**
+ * WallCreateContentMenu is the widget for Menu above wall create content Form
+ *
+ * @property-read ContentContainerActiveRecord $contentContainer
+ * @author luke
+ * @since 1.13.0
+ */
+class WallCreateContentMenu extends Menu
+{
+    /**
+     * @inheritdoc
+     */
+    public $id = 'contentFormMenu';
+
+    /**
+     * @inheritdoc
+     */
+    public $jsWidget = 'content.form.CreateFormMenu';
+
+    /**
+     * @inheritdoc
+     */
+    public $init = true;
+
+    /**
+     * @inheritdoc
+     */
+    public $template = 'wallCreateContentMenu';
+
+    /**
+     * @var int What first menu entries should be visible as tabs top level, rest entries will be grouped into sub-menu
+     */
+    public $visibleEntriesNum = 2;
```

Comment:
Wouldn't it be better to implement this via Javascript, and decide there how many entries are displayed as tabs and which ones in the menu?

Then you could react better to mobile or different resolutions.

But this is just a note, maybe this can be implemented/changed later.